Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -64,7 +64,23 @@ The Android NDK contains prebuilt versions of the native compiler and toolchain.
 
 Besides picking the right architecture, you need to specify the correct sysroot for the native API level you want to target. The sysroot is a directory that contains the system headers and libraries for your target. Native APIs vary by Android API level. Possible sysroots for each Android API level are in `$NDK/platforms/`. Each API level directory contains subdirectories for the various CPUs and architectures.
 
-One possibility for setting up the build system is exporting the compiler path and necessary flags as environment variables. To make things easier, however, the NDK allows you to create a so-called standalone toolchain—a "temporary" toolchain that incorporates the required settings.
+One possibility for setting up the build system is exporting the compiler path and necessary flags as environment variables. To make things easier, however, the NDK allows you to create a so-called standalone toolchain — a "temporary" toolchain that incorporates the required settings.
+
+The local NDK installation can be managed through Android Studio. Select "Configure->SDK Manager" from the initial Android Studio window, or create an empty project in Android Studio and select "Tools->Android->SDK Manager" to open the SDK Manager GUI. The "SDK Tools" tab lets you install the NDK.
```

Comment:
This is the same as in line 27. Please use this paragraph only once.

Reply:
It is almost the same, but in this case it references the NDK, not the SDK, although both are accessed via the SDK Manager, hence the repetition.